sdk/monitor/azure-monitor-opentelemetry/CHANGELOG.md

```diff
@@ -1,14 +1,6 @@
 # Release History
 
-## 1.0.0b17 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+## 1.0.0 (2023-09-05)
```

Comment:
Maybe a one-liner like "Stable release." here would do the trick?
Actually, I should probably add a note to the changelog that the subnamespaces have been made private.

Reply:
Also - release date is for today? Official release week in next week...

Reply:
Oh I wasn't sure what to put for the date. Would it be the "all package ship date", which is 9/12 instead?